### ISSUE TITLE
Incompatibility with amsgen

### ISSUE THREAD

**Issue by skafdasschaf** [skafdasschaf/latex-pgfgantt#4]
as proposed by Scott Pakin on 26.07.2017

"... I found that pgfgantt is incompatible with the amsgen package, which is loaded by amsmath, which is loaded by numerous other packages.  For complete details and a sample LaTeX file that showcases the problem, see the comp.text.tex thread, "Conflict between mathtools and pgfgantt":

    https://groups.google.com/forum/#!topic/comp.text.tex/6GFVD_iC__I

(I at first thought mathtools was the culprint, hence the name of the
thread.)

In short, amsgen redefines \@ifstar, and that confuses pgfgantt.  I've attached an updated pgfgantt that explicitly uses LaTeX2e'e original \@ifstar (or rather, something equivalent to it) to make the package robust to amsgen's redefinition. ... "
[pgfgantt.tar.gz](https://github.com/skafdasschaf/latex-pgfgantt/files/1209452/pgfgantt.tar.gz)

**Comment by skafdasschaf** [comment]
Has been fixed in v5.0
